concepticondata/references/references.bib

```diff
@@ -3221,3 +3221,20 @@ @book{LaPolla2003
   volume       = {31},
   year         = {2003}
 }
+
+@phdthesis{Josserand1983,
+	address = {New Orleans},
+	author = {Josserand, Judy Kathryn},
+	school = {Tulane University},
+	title = {Mixtec dialect history},
+	year = {1983}
+}
+
+@article{Durr1987,
+	author = {D{\"u}rr, Michael},
+	journal = {Indiana},
+	pages = {19--61},
```

Comment:
Thanks for updating the file @SAuderset! Just a tiny stylistic thing that is not so obvious. We write the page numbers with a single hyphen: "19-61". Would you mind changing this?

Reply:
oh, I had no idea! sure, I changed it.